scripts/script.js

```diff
@@ -0,0 +1,186 @@
+const container = document.querySelector(".container");
+const display = document.getElementById("display");
+const historyList = document.getElementById("history");
+const memoryList = document.getElementById("memory");
+
+let memory = 0;
+let history = [];
+let string = "";
+
+container.addEventListener("click", (e) => {
+  const button = e.target;
+
+  const value = button.value;
```

Comment:
no need of creating this another variable for value you can directly get button value see below
const buttonValue = e.target.value;